src/components/animated/_pudding.less

```diff
@@ -0,0 +1,35 @@
+.pudding-btn() {
```

Comment:
The problem is here. When you add `()` in less it creates a mixin not a class. Remove them to get it to work

Reply:
Of couse! How stupid of me